### PR TITLE
Newcustomlogin, About Page update

### PR DESCRIPTION
**Pull request by simonhlee97**
previous PR attempt failed. Another attempt.
About Page updated with contributors and some styling.

### CODE REVIEW

File: apps/web/src/components/Member/index.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+interface MemberProps {
+  name: string;
+  username: string;
+}
+
+export const Member: React.FC<MemberProps> = ({ name, username }) => {
+  return (
+    <StyledMember>
+      <Card>
+        <H1>{name}</H1>
```
Comment (ivanms1) [inline apps/web/src/components/Member/index.tsx:14]: could we have a more meaninful name for this? like `<Title />` or something similar

File: apps/web/src/components/Member/index.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link';
+
+import { StyledMember, Card, ProfileImage, H1 } from './styles';
+
+interface MemberProps {
+  name: string;
+  username: string;
+}
+
+export const Member: React.FC<MemberProps> = ({ name, username }) => {
```
Comment (ivanms1) [inline apps/web/src/components/Member/index.tsx:10]: This is correct but just for consistency, could you add the types like this?
```suggestion
export const Member = ({ name, username } : MemberProps) => {
```

File: apps/web/src/components/Members/index.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  {
+    id: 3,
+    name: 'Mina',
+    username: 'jm066',
+  },
+];
+
+const Members = () => {
+  return (
+    <>
```
Comment (ivanms1) [inline apps/web/src/components/Members/index.tsx:26]: this is not necessary

File: apps/web/src/pages/About/index.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import Members from '@/components/Members';
+import React from 'react';
+import { StyledAbout } from './styles';
+
+const About = () => {
+  return (
+    <>
```
Comment (ivanms1) [inline apps/web/src/pages/About/index.tsx:7]: not necessary, please remove.

File: apps/web/src/pages/Signin/index.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import { signIn } from 'next-auth/react';
+import { StyledSignin, LoginButton, LoginContent, Header } from './styles';
+
+interface SigninProps {
+  providers: { id: string; name: string }[];
+}
+
+const Signin = ({ providers }: SigninProps) => {
+  const onSubmit = (data) => alert(JSON.stringify(data, null, 2));
```
Comment (ivanms1) [inline apps/web/src/pages/Signin/index.tsx:9]: this is not being used?

File: apps/web/src/pages/Signin/styles.ts [hunk 6]
```diff
@@ -0,0 +1,29 @@
+import { Button } from 'ui';
+import { css, styled } from '@/stitches/stitches.config';
+export const Header = styled('h1', {
```
Comment (ivanms1) [inline apps/web/src/pages/Signin/styles.ts:3]: please add a space for readability

**Review by ivanms1 — CHANGES_REQUESTED** [review]
Looks good, just some small comments.

**Comment by ivanms1** [comment]
Task linked: [CU-29bkqcx Customize the Login page. right now, it's just the NextAuth default screen](https://app.clickup.com/t/29bkqcx)

**Comment by ivanms1** [comment]
Task linked: [CU-27r0mvd public route "/about" page](https://app.clickup.com/t/27r0mvd)

File: apps/web/src/components/Members/index.tsx (hunk 3 above)
Comment (simonhlee97) [reply apps/web/src/components/Members/index.tsx:26]: can you elaborate?

File: apps/web/src/pages/About/index.tsx (hunk 4 above)
Comment (simonhlee97) [reply apps/web/src/pages/About/index.tsx:7]: `const About = () => (
  <StyledAbout>
    <Members />
  </StyledAbout>
);`

i changed the file to this.

File: apps/web/src/pages/Signin/index.tsx (hunk 5 above)
Comment (simonhlee97) [reply apps/web/src/pages/Signin/index.tsx:9]: I commented out the onSubmit line.

File: apps/web/src/components/Members/index.tsx (hunk 3 above)
Comment (ivanms1) [reply apps/web/src/components/Members/index.tsx:26]: you can remove the `<></>`

File: apps/web/src/pages/Signin/index.tsx (hunk 5 above)
Comment (ivanms1) [reply apps/web/src/pages/Signin/index.tsx:9]: ah ok, could you remove then? if it's not being used.

File: apps/api/src/graphql/Project.ts [hunk 7]
```diff
@@ -34,8 +34,8 @@ export const ProjectType = objectType({
       description: 'If this project is liked by the current user',
       async resolve(_root, _, ctx) {
         try {
-          const currentUserId = decodeAccessToken(ctx.accessToken);
-          if (!currentUserId) {
+          const currentUserId = decodeAccessToken(ctx?.accessToken);
+          if (currentUserId) {
```
Comment (ivanms1) [inline apps/api/src/graphql/Project.ts:38]: could you check this line back to `!currentUserId`

File: apps/web/src/components/Members/index.tsx (hunk 3 above)
Comment (ivanms1) [reply apps/web/src/components/Members/index.tsx:31]: please remove

**Review by ivanms1 — COMMENTED** [review]
Just last two changes and ready to merge

File: apps/api/src/graphql/Project.ts (hunk 7 above)
Comment (simonhlee97) [reply apps/api/src/graphql/Project.ts]: done.

**Review by ivanms1 — APPROVED** [review]
LGTM

**Pull request merged**